backend/utility/emailTemplates.js

```diff
@@ -0,0 +1,13 @@
+// This function will be used to generate a template which will be utilized in all email types
+// module.exports.generateHtml(data){};
+
+module.exports.getMailTemplate = (type, data) => {
+  const templates = {
+    'INVITE-ADMIN': {
+      // html:"",
+      subject: '[INVITATION] Invited to become an admin',
+      text: `Hi ${data.name}!\n We would love for you to join us at HITK Tech Community as an admin.\n\tHITK Tech Community is a platform built by the students and for the students with the main intent of increasing awareness towards plethora of opportunities and internships in tech all around/over the year. This will not only give practical work experience/exposure to students, but will also help everyone to know and grab their required opportunities in time!\nClick on ${data.link} for further action`,
```

Comment:
Can you suggest an approach? 

The best one I could find was alex's answer here - https://stackoverflow.com/questions/7975005/format-a-javascript-string-using-placeholders-and-an-object-of-substitutions

Implementing this might require some additional refactoring.

Reply:
I would encourage you to think on it a little more. This issue is stuck just for that.
A little refactoring with a few assumptions might work(haven't tried this yet). I did similar thing before using some alternate means